### PR TITLE
Improve code-push command

### PR DESCRIPTION
**Pull request by gmbharath12**
- Fixes https://github.com/electrode-io/electrode-native/issues/299

### CODE REVIEW

File: ern-local-cli/src/lib/publication.js [hunk 1]
```diff
@@ -283,7 +283,7 @@ miniApps: Array<Dependency>, {
   }
 
   const pathsToMiniAppsToBeCodePushed = _.map(miniAppsToBeCodePushed, m => DependencyPath.fromString(m.toString()))
-  await spin('Generating composite bundle to be published through CodePush',
+  await spin('Generating MiniApps composite',
```
Comment (krunalsshah) [inline ern-local-cli/src/lib/publication.js]: composite miniapps - change the order

File: ern-local-cli/src/lib/publication.js [hunk 2]
```diff
@@ -293,25 +293,25 @@ miniApps: Array<Dependency>, {
     ? path.join(bundleOutputDirectory, 'index.android.bundle')
     : path.join(bundleOutputDirectory, 'MiniApp.jsbundle')
 
-  await reactnative.bundle({
+  await spin('Generating MiniApps composite bundle', reactnative.bundle({
```
Comment (krunalsshah) [inline ern-local-cli/src/lib/publication.js]: composite bundle for miniapps

**Comment by krunalsshah** [comment]
@gmbharath12 flawless 💯

**Comment by gmbharath12** [comment]
@krunalsshah Many thanks to you *Sir* and *King* @belemaire. Your blessings helping me !

**Pull request merged**